server/models/index.js

```diff
@@ -41,21 +43,41 @@ let saveWishlist = async (products, username) => {
   return await instance.save();
 };
 
-let getProducts = async () => {
-  return await MyProductsModel.find({}).sort([['productId', 'ascending']]).exec();
+let getProducts = (callback) => {
+  MyProductsModel.find({}).sort([['productId', 'ascending']]).exec(function (err, docs) {
+    callback(err, docs);
+  });
 };
-
-let getWishlists = async () => {
-  return await MyWishlistModel.find({});
+let getWishlists = (callback) => {
+  MyWishlistModel.find({}, (err, docs) => {
+    callback(err, docs);
+  });
 };
 //get product by id
-let getProductById = async (productId) => {
-  return await MyProductsModel.find({productId: productId});
+let getProductById = (productId, callback) => {
+  MyProductsModel.find({productId: productId}, (err, docs) => {
+    callback(err, docs);
+  });
 };
 //get wishlist by username
-let getWishlistByUsername = async (username) => {
-  return await MyWishlistModel.find({username: username});
+let getWishlistByUsername = (username, callback) => {
+  MyWishlistModel.find({username: username}, (err, docs) => {
+    callback(err, docs);
+  });
+};
+//update product when liked
+let updateProduct = (productId, like, callback) => {
+
+  MyProductsModel.updateOne({productId: productId}, {like: like}, (err, docs) => {
+    try {
+      return docs;
```

Comment:
This `return` statement doesn't do anything. We should be invoking the `callback`.